docs/howto/test.md

```diff
@@ -0,0 +1,78 @@
+# How to test
+
+## Test helpers
```

Comment:
This is an end-user's guide to testing, right? (vs. in CONTRIBUTING.md how to develop on AMS).

In any case, this is :100: